classes/03class/exercises/c03-docker05/mfreitassm/README.md

```diff
@@ -0,0 +1,50 @@
+# C03-Docker05
+
+## Command Execution Output
+- Commands executed to run both containers:
+```
+docker run -dp 8081:80 mfreitassm/devopsacademy:v1
+Unable to find image 'mfreitassm/devopsacademy:v1' locally
+v1: Pulling from mfreitassm/devopsacademy
+aad63a933944: Pull complete 
+29ade582b51e: Pull complete 
+7e41ad5b6f9c: Pull complete 
+ebf61b47b4ca: Pull complete 
+9c060bce4eae: Pull complete 
+d6dd71732fec: Pull complete 
+Digest: sha256:26f917769736a9257687038ba760c2ad2cc7ecf902a9f8d72eb111ebc773f452
+Status: Downloaded newer image for mfreitassm/devopsacademy:v1
+780676bbe522bbdec6b4fdd1471bef37af490c6573c25750d83cb59efc5141d0
+
+docker run -dp 8082:80 mfreitassm/devopsacademy:v2
+Unable to find image 'mfreitassm/devopsacademy:v2' locally
+v2: Pulling from mfreitassm/devopsacademy
+aad63a933944: Already exists 
+29ade582b51e: Already exists 
+7e41ad5b6f9c: Already exists 
+ebf61b47b4ca: Already exists 
+9c060bce4eae: Already exists 
+6e9b91bc1a0e: Pull complete 
+Digest: sha256:0c1e6acfaab9f6216af2773f168dbb7faa498ce741886fa94bde0866baa1af4e
+Status: Downloaded newer image for mfreitassm/devopsacademy:v2
+620554f64e541916dd63f8fdb6c70b54887a78dfd9aa3ad964b1aa76999e91f9
+docker: Error response from daemon: driver failed programming external connectivity on endpoint determined_chatelet (2e546e12d9e44bcfe5badb01a8a6a00eac704505f68bbdf2bbafa8e065be88b2): Bind for 0.0.0.0:8081 failed: port is already allocated.
```

Comment:
Looking good!. 
Just curious about what happened after this line? port 8081 was allocated for your V1 container but you are running the V2 container again.

Reply:
Any update on this @mfreitassm ? :)